Apigee/Mint/Organization.php

```diff
@@ -699,12 +702,10 @@ public function getPrepaidBalanceReport($month, $year, $developer_id, $currency_
                 'billingYear' => $year
             );
 
-            $url = '/mint/organizations/' . rawurlencode($this->config->orgName) . '/prepaid-balance-reports';
+            $url = rawurlencode($this->config->orgName) . '/prepaid-balance-reports';
```

Comment:
Did you mean to change the value of `$url` here? If so, an explanation would help.

Reply:
Here:
```
$base_url = '/mint/organizations';
```
which makes it unnecessary to change base url, we can just append path 
```
rawurlencode($this->config->orgName) . '/prepaid-balance-reports';
```
to the existing base URL, which means passing as first parameter of `post()` request everything that we want to attach to `$base_url`